mne/channels/tests/test_montage.py

```diff
@@ -1458,4 +1458,29 @@ def test_plot_montage():
     plt.close('all')
 
 
+@testing.requires_testing_data
+def test_montage_add_estimated_fiducials():
+    data_path = testing.data_path(download=True)
+
```

Comment:
It's not a good idea to add `download=True` in tests. And in any case this is already defined up on line 48 (or so), so:
```suggestion
```